src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -77,6 +89,21 @@ String servicePointUserFilter(String s) {
     return res;
   }
 
+  @Override
+  public void postTenant(TenantAttributes tenantAttributes, Map<String, String> headers, Handler<AsyncResult<Response>> handler, Context context) {
```

Comment:
I thought mod-inventory-storage also initialised some Kafka publishing when a tenant is enabled, do this happen elsewhere?

Reply:
It was before postTenant, but then it was moved to loadData by folks.

Reply:
Thanks.

Reply:
Do you know why it was moved to `loadData`? Was it to only have one of either `loadData` or `postTenant`?

Reply:
No, I don't know. Maybe loadData is designed for such thinks, but again it doesn't work for me unfortunately

Reply:
The postTenant is removed, loadData started to work for me.